sections/shorting/ShortingStats/ShortingStats.tsx

```diff
@@ -64,7 +75,7 @@ const ShortingStats = () => {
 
 	const total = useMemo(() => {
 		if (shortStatsMap != null) {
-			return reduce(shortStatsMap, (sum, short) => sum.plus(short.openInterest), zeroBN);
+			return reduce(shortStatsMap, (sum, short) => sum.add(short.openInterest), zeroBN);
```

Comment:
is `zeroBN` correct here?

Reply:
I believe so, we are basically taking a sum of all the open interest items on the shorts here, right? so  `zeroBN` is the starting value (`zeroBN` is actually `wei(0)`)